DOCUMENTATION.md

```diff
@@ -2552,142 +2573,175 @@ console.log(value);
 
 The SQL service provided by Hazelcast Node.js client allows you to run SQL queries.
 
-> **WARNING: The SQL feature have become stable in 5.0. In order a client and a server to be fully compatible with each other,
-> their major versions must be the same.**
+You can use SQL to query data in maps, Kafka topics, or a variety of file systems. Results can be sent directly to the client
+or inserted into maps or Kafka topics. For streaming queries, you can submit them to a cluster as jobs to run in the background.
 
-> **WARNING: A [Unisocket Client](#822-unisocket-client) must connect to a member that is not
-> [lite](https://docs.hazelcast.com/hazelcast/latest/management/cluster-utilities.html#enabling-lite-members) (non-lite members
-> are called data members), otherwise Data Manipulation Language(DML) statements below won't work for that client.**
+> **WARNING: The SQL feature have become stable in 5.0 versions of the client and the server. In order a client and a server to
+> be fully compatible with each other, their major versions must be the same.**
 
-### 8.7.1 SQL Statements
+> **WARNING:  To use the SQL feature, the Jet engine must be enabled on the members and the `hazelcast-sql` module must be in the
+> classpath  of the members. If you are using the CLI, Docker image, or distributions to start Hazelcast members, then you don't
+> need to  do anything, as the above preconditions are already satisfied for such members.**
+>
+> **However, if you are using Hazelcast members in the embedded mode, or receiving errors saying that
```

Comment:
Is the `**` before `However` necessary?

Reply:
Yeah it was necessary. Since there is a newline in between these